app/scripts/controllers/membership.js

```diff
@@ -27,7 +27,7 @@ angular
 
       // NOTE: these could all be moved out into a strings service.
       var messages = {
-        errorReason: _.template('Reason: "<%- httpErr %>"'),
+        errorReason: _.template('Reason: "<%= httpErr %>"'),
```

Comment:
Here is the difference, for reference:

Escaping error messages from server:
![screen shot 2016-10-25 at 9 50 43 am](https://cloud.githubusercontent.com/assets/280512/19688658/9af80d44-9a98-11e6-8923-94e63c527e15.png)

Not escaping messages from server:
![screen shot 2016-10-25 at 9 51 16 am](https://cloud.githubusercontent.com/assets/280512/19688657/9af5752a-9a98-11e6-91dd-e9a8a58ad3b8.png)